src/routes/+layout.svelte

```diff
@@ -140,6 +140,11 @@
 					<span class="font-bold tracking-wide"> 2023 </span>
 				</div>
 			</li>
+			<li>
+				<div class="">
```

Comment:
you forgot to add this to the mobile nav also, some lines above (can you add also a link for the blog in both places? /blog is the href)